docs/require-login.md

```diff
@@ -1,8 +1,18 @@
 # Require Login
 
-By default, all websites are publicly accessible. In some situations, you may want to require users be logged in to access the website. This is especially useful when in pre-launch mode.
+By default, all websites are publicly accessible. In some situations, you may want to require users to be logged in to access the website. This is especially useful when in pre-launch mode.
 
-Environments running in Cloud that are not of type `production` have the `require-login` feature enabled by default. Set the `security.require-login` setting to `true` to require all users be logged in to view the website. If you want to make a non-production site public, set the `altis.environments.${ environment-name }.modules.security.require-login` setting to `false`.
+## Multisite Environments
+
+In a multisite environment, requiring login is as easy as unchecking the site's public setting in the Edit Site screen. To access this setting, go to My Sites > Network Admin > Sites > Click the URL for the site you want to edit > Attributes > Public.
```

Comment:
Fun fact, you can link directly to the appropriate section of the admin (and other places) using a special markdown feature in our docs:  

https://www.altis-dxp.com/resources/docs/documentation/linking/#internal-linking

So this could be

```suggestion
In a multisite environment, requiring login is as easy as unchecking the site's public setting in the Edit Site screen. To access this setting, go to [My Sites > Network Admin > Sites](internal://network-admin/sites.php) and the click the URL for the site you want to edit. From there you check the box for whether the site is public or not under the "Attributes" section.
```

Reply:
✨